{{cookiecutter.__src_folder_name}}/requirements-dev.in

```diff
@@ -17,5 +17,4 @@ pytest-flask
 pytest-playwright
 
 # Linters
-ruff
-black
+ruff
```

Comment:
EOF newline changed.

We do have a precommit that fixes EOF newlines in the root of this project, I believe.